src/styles/index.css

```diff
@@ -1,4 +1,5 @@
 @import url('https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500;600;800;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
```

Comment:
we use Rubick accross app, remove this

Reply:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/97129041/24df7871-29cd-460b-8fdc-c2c08b42c82b)
Yes, no need to put another font for slashes

Reply:
Ok, now it will look like this. Is this ok ?
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/89192246/04d99441-bb4c-4d55-96eb-1292ca350735)